UnityProject/Assets/Scripts/Objects/Medical/CloningPod.cs

```diff
@@ -15,8 +18,19 @@ public class CloningPod : NetworkBehaviour
 		public string statusString;
 		public CloningConsole console;
 
+		[FormerlySerializedAs("LimbCloningDamage"), SerializeField] private float internalLimbCloningDamage = 25;
+
+		[SerializeField] private float internalCloningTime = 60;
+
+
 		public float LimbCloningDamage = 25;
 
+		public float CloningTime = 60;
```

Comment:
why are both ``internalCloningTime`` and ``CloningTime`` serialized? Maybe you want the later to be a property?